README.md

```diff
@@ -1 +1,129 @@
-# scalardb-samples
\ No newline at end of file
+# Scalar DB Samples
```

Comment:
```suggestion
# Scalar DB Server Samples
```
Since this sample targets Scalar DB Server and not Scalar DB native. Should the title reflect that to avoid any confusion even though it is explained below ?